core/src/test/scala/integration/kafka/api/MetricsTest.scala

```diff
@@ -323,7 +303,7 @@ class MetricsTest extends IntegrationTestHarness with SaslSetup {
     }
   }
 
-  private def yammerMeterWithPrefix(prefix: String): Meter = {
+  def yammerMeterWithPrefix(prefix: String): Meter = {
```

Comment:
nit: Any reason for making this non-private?  I found this not used anywhere else in this PR

Reply:
Good catch. Fixed.